### DISCUSSION TITLE
Image Class: Hasil Submission Berbeda

### DISCUSSION THREAD

**Discussion by Githubmuls** [teamalgoritma/community#342] [Q&A] [answered]
Saya melakukan pembuatan file submission sesuai dengan step yang diberikan, namun terdapat perbedaan output. Terdapat penambahan kolom/variabel "X" namun hanya terlihat ketika file submissionnya dibaca kembali menggunakan read.csv.

> Membuat File Submission

![image](https://user-images.githubusercontent.com/107347991/192089789-9469c11f-a2ad-4a45-8ab0-3579247dddf2.png)

> Jika filenya dibaca kembali dan dibandingkan dengan file contoh ada perbedaan jumlah variabel (penambahan X)

![image](https://user-images.githubusercontent.com/107347991/192090134-98b76d9b-00b0-4055-b070-56db8691e466.png)

![image](https://user-images.githubusercontent.com/107347991/192090154-67f2a0fc-0d7f-47a6-bddf-a7ab48f4c517.png)

**Comment by fafilia** [comment, marked as answer]
Dear @Githubmuls ,

Untuk mengatasi hal tersebut, silahkan tambahkan parameter row.names=FALSE saat write csv seperti contoh berikut:
```
write.csv(test, "submission,csv", row.names = FALSE)
```

Demikian yang dapat saya sampaikan, semoga bermanfaat.
